.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: Build
+
+on:
+  pull_request:
+    branches: ['**']
+  push:
+    branches: ['**']
+    tags: [v*]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build and Push Docker Image
+        uses: Dwolla/jenkins-agents-workflow/.github/actions/build@main
```

Comment:
I wonder if we should be using a more explicit version than just `@main`—could tie into updating the workflow to resolve all the warnings I mentioned.